blocks/header/header.js

```diff
@@ -91,6 +91,15 @@ function toggleMenu(nav, navSections, forceExpanded = null) {
  * @param {Element} block The header block element
  */
 export default async function decorate(block) {
+  // Global nav
+  const json = document.createElement('script');
+  json.appendChild(document.createTextNode('window.gnav_jsonPath = 20220-nav-config.25.json'));
+  block.appendChild(json);
+
+  const script = document.createElement('script');
+  script.src = 'https://webapps-cdn.esri.com/CDN/components/global-nav/js/gn.js';
+  block.appendChild(script);
+
```

Comment:
Can you remove the rest of the header code? That's just boilerplate that's only needed if we're implementing the nav in-project.